source/ja/fv_convert.rst

```diff
@@ -932,4 +935,200 @@ binary_typesで指定できる。`OpenCV <https://github.com/opencv>`_ を利用
+ ``--enable-python-plugin`` または ``--enable-python3-plugin`` オプション付きでコンパイルした場合のみ利用可能である。
+ バイナリパッケージ (RHEL 6 を除く) でもこのプラグインは利用可能である。
+
+  :function:  上記のインタフェース名を指定する (例: ``string_feature``)。
+  :module:    Pythonモジュール名を指定する。
+  :class:     Pythonモジュール内のクラス名を指定する。
+
+ .. code-block:: js
+
+      "binary_types": {
+        "extract_length": {
+          "method": "dynamic",
+          "path": "libpython_bridge.so",
+          "function": "binary_feature",
+          "module": "binary_length",
+          "class": "BinaryLengthExtractor"
+        }
+      }
+
+上記に加えて、Pythonクラスに渡す任意のパラメタを指定することができる。
+パラメタのキーとバリューは共に文字列型である必要がある。
+
+Python モジュールに対する共通要件
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Python で特徴抽出ロジックを実装する場合、Pythonクラスをモジュールに定義する必要がある。
+以下に例を示す。
+
+.. code-block:: python
+
+  class MyFeatureExtractor(object):
+      @classmethod
+      def create(cls, param):
+          return cls()
+
+      def extract(self, key, value):
+          return [(u'key1', 1.0), (u'key2', 2.0)]
+
+Pythonクラスに対する要件を以下に示す。
+
+* クラスは ``create`` クラスメソッドを実装する必要がある。
+  引数 ``params`` は、設定ファイルで指定されたパラメタ (``dict`` 型) である。
+* 各クラスは、各インタフェースで要求されているシグネチャのインスタンスメソッドを実装する必要がある。
+  以下の各セクションを参照のこと。
+* モジュールは、 ``sys.path`` 内のいずれかのパス、 ``PYTHONPATH`` 環境変数内のいずれかのパス、またはデフォルトの Python モジュールディレクトリ (``$PREFIX/lib/jubatus/python``) のいずれかに配置されている必要がある。
+  バイナリパッケージでインストールした Python Bridge は ``pyenv`` を認識しないため注意すること。
+  ``pyenv`` でインストールした Python パッケージを利用したい場合は、Jubatus プロセスを開始する前に ``PYTHONPATH`` を定義すること (例: ``export PYTHONPATH="$HOME/.pyenv/versions/3.5.1/lib/python3.5/site-packages"``)。
+
+Python コードでエラーが発生した場合は標準エラー出力に出力される。
+
+以下の各セクションでは、Python 3.x の型名で説明を行う。
+Python 2.x を使用している場合は、 ``str`` と ``bytes`` は ``unicode`` と ``str`` にそれぞれ読み替えること。
+
+``string_feature`` インタフェース
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+クラスにインスタンスメソッド ``extract`` (1 引数) を定義する必要がある。
+
+* 引数は入力のテキストである (``str`` 型)。
+* 戻り値は 4 要素のタプルのリストである。タプルの各要素の意味は以下の通りである。
+
+    * 抽出したデータの開始位置 (``int`` 型); ``0`` でも良い。
+    * 抽出したデータの長さ (``int`` 型); ``0`` でも良い。
+    * 抽出した文字列データ (``str`` 型)。
+    * 抽出した文字列データに対するスコア (``float`` 型); 通常は ``1.0`` を使用すれば良い。
+
+サンプルとして、英語の文章に対するステミング (Porter Stemmer) を行う実装がデフォルトで提供されている。
+詳細は `sentence_stemmer モジュール <https://github.com/jubatus/jubatus/blob/master/plugin/src/fv_converter/python_bridge/python/sentence_stemmer.py>`_ のソースを参照すること。
+このサンプルを動作させるには `Natural Language Toolkit <http://www.nltk.org/>`_ のインストールが必要である (``pip install nltk``)。
+
+ .. code-block:: js
+
+      "string_types": {
+        "stem_sentence": {
+          "method": "dynamic",
+          "path": "libpython_bridge.so",
+          "function": "string_feature",
+          "module": "sentence_stemmer",
+          "class": "SentenceStemmer"
+        }
+      },
+      "string_rules": [
+        { "key": "*", "type": "stem_sentence", "sample_weight": "tf", "global_weight": "bin" }
+      ]
+
+``word_splitter`` インタフェース
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+クラスにインスタンスメソッド ``split`` (1 引数) を定義する必要がある。
```

Comment:
`word_splitter` interface needs to implement not `extract` but `split` method and it is bit confusing.
If you agree, we should write python implementation example like:

```
class MyWordSplitter(object):
    @classmethod
    def create(cls, param):
         return cls()

    def split(self, text):
        return text.split(',')
```